xskillscore/core/comparative.py

```diff
@@ -1,10 +1,14 @@
+import inspect
 import warnings
+from typing import List, Mapping, Optional, Tuple, Union
 
 import numpy as np
 import scipy.stats as st
 import xarray as xr
 
-from .deterministic import mae, pearson_r
+from . import deterministic as dm
+
+XArray = Union[xr.Dataset, xr.DataArray]
```

Comment:
nice you are using typing, we want to add that #317

Reply:
I actually saw that when I was deciding to add typing.